### PR TITLE
Team Service and Readme update

### PR DESCRIPTION
**Pull request by partamonov**
Team Service to resolve https://github.com/russellcardullo/go-pingdom/issues/24

### CODE REVIEW

**Comment by partamonov** [comment]
@russellcardullo could you review?

File: pingdom/api_responses.go [hunk 1]
```diff
@@ -82,6 +82,25 @@ type ProbeResponse struct {
 	Region     string `json:"region"`
 }
 
+// TeamResponse represents the json response for teams from the PIngdom API
+type TeamResponse struct {
+	ID    interface{} `json:"id"`
```
Comment (russellcardullo) [inline pingdom/api_responses.go]: The Pingdom documentation is inconsistent on whether IDs in team responses are integers or strings.  The documentation for (`Get Teams`, `Get Team`) show the ID as a string while (`Create Team`, `Update Team`) show the ID as an int.

However when I try this all 4 methods return a string for me.  Do you see that also?

File: pingdom/team.go [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	t := &TeamResponse{}
+	_, err = cs.client.Do(req, t)
+	if err != nil {
+		return nil, err
+	}
+	return t, err
+}
+
+// Delete will delete the Team for the given ID.
+func (cs *TeamService) Delete(id int) (bool, error) {
```
Comment (russellcardullo) [inline pingdom/team.go]: Would you mind making this return `(*PingdomResponse, error)` to be consistent with the other service APIs for `Delete`?

File: pingdom/team_types_test.go [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+func TestTeamValid(t *testing.T) {
+	team := TeamData{
+		Name:    "fake team",
+		UserIDs: "",
+	}
+	params := team.Valid()
+
+	assert.Equal(t, nil, params, "Maintenance.Valid() should return nil if valid")
```
Comment (russellcardullo) [inline pingdom/team_types_test.go]: Could you change the message here from `Maintenance` to `Team`?

File: pingdom/team_types.go [hunk 4]
```diff
@@ -0,0 +1,50 @@
+package pingdom
+
+import (
+	"fmt"
+)
+
+// TeamData represents a Pingdom Team Data.
+type TeamData struct {
+	Name    string `json:"name"`
+	UserIDs string `json:"userids,omitempty"`
```
Comment (russellcardullo) [inline pingdom/team_types.go]: Elsewhere in the other types we're calling this `UserIds`.  Could we make that the same here?  I think it'd be better for it to be consistent throughout the types.

File: pingdom/api_responses.go (hunk 1 above)
Comment (partamonov) [reply pingdom/api_responses.go]: I think I saw int in return, but I will re-check.

File: pingdom/team.go (hunk 2 above)
Comment (partamonov) [reply pingdom/team.go]: It is different structure, but made is as `(*TeamDeleteResponse, error)`

File: pingdom/team_types_test.go (hunk 3 above)
Comment (partamonov) [reply pingdom/team_types_test.go]: Done

File: pingdom/team_types.go (hunk 4 above)
Comment (partamonov) [reply pingdom/team_types.go]: Done

**Comment by partamonov** [comment]
@russellcardullo done with your remarks

**Pull request merged**